tests/acceptance/repo/settings-test.js

```diff
@@ -264,7 +277,42 @@ module('Acceptance | repo settings', function (hooks) {
 
     assert.equal(deletedIds.pop(), this.dailyCron.id, 'expected the server to have received a deletion request for the first cron');
     assert.equal(settingsPage.crons.length, 1, 'expected only one cron to remain');
-    done();
+
+    let requestBodies = [];
+
+    server.post(`/repo/${this.repository.id}/branch/another-branch/cron`, (schema, request) => {
+      let newCron = server.create('cron', {
+        interval: 'daily',
+        dont_run_if_recent_build_exists: false,
+        repository: this.repository
+      });
+
+      let parsedRequestBody = JSON.parse(request.requestBody);
+      requestBodies.push(parsedRequestBody);
+
+      return newCron;
+    });
+
+    await settingsPage.cronForm.branch.click();
+    assert.equal(settingsPage.cronForm.branch.options.length, 2);
+
+    await settingsPage.cronForm.branch.search('another');
+    // FIXME the options aren’t what I would expect…
+    // assert.equal(settingsPage.cronForm.branch.options.length, 1);
```

Comment:
My desire with these assertions is to first test what shows before any search has happened (the current plan is to show branches sorted in alphabetic order) and then show the results of a search. Looking at the Mirage output, the search _is_ happening, but the popup element doesn’t contain the results I’d expect. It’s strange because the code does seem to work in production.